coredb-operator/src/apis/coredb_types.rs

```diff
@@ -22,6 +22,10 @@ pub struct CoreDBSpec {
     pub resources: ResourceRequirements,
     #[serde(default = "defaults::default_storage")]
     pub storage: Quantity,
+    #[serde(default = "defaults::default_sharedir_storage")]
+    pub sharedir_storage: Quantity,
+    #[serde(default = "defaults::default_pkglibdir_storage")]
+    pub pkglibdir_storage: Quantity,
```

Comment:
Nit - These are typically camelCase (`postgresExporterImage` and `apiVersion` for example).